arch/arm/core/cortex_m/Kconfig

```diff
@@ -194,6 +194,7 @@ config SW_VECTOR_RELAY
 	bool
 	prompt "Enable Cortex-M0 Vector Table soft relay"
 	default n
+	default y if BOOTLOADER_MCUBOOT
```

Comment:
Please note that the `depends` below will ensure that this option is *not* enabled if building with `BOOTLOADER_MCUBOOT` for a target that is not a Cortex-M0